concordia/static/js/base.js

```diff
@@ -73,10 +73,11 @@ function displayMessage(level, message, uniqueId) {
 }
 
 function isOutdatedBrowser() {
+    /* See https://caniuse.com/#feat=css-supports-api */
     if (typeof CSS == 'undefined' || !CSS.supports) {
         return true;
     }
-    return !CSS.supports('display: flex');
+    return false;
```

Comment:
Since we're changing this to use only a single check we could replace the function body with a simple `return (typeof CSS == 'undefined' || !CSS.supports);`